apps/bfd-server/bfd-server-war/src/test/resources/openapi/posts.yaml

```diff
@@ -354,3 +354,241 @@ paths:
+                      '-', with a length limit of 11 characters.
+                      ```
+
+                      The following are all valid values for MBI, and all might
+                      represent the same resource:
+                        - `mbi=9AB2WW3GR44` (unhashed MBI)
+                        - `mbi=82273caf4d2c3b5a8340190ae3575950957ce469e593efd7736d60c3b39d253c` (hashed)
+                        - `mbi=1S00E00HA26` (synthetic MBI)
+                    example: "9AB2WW3GR44"
+                  _lastUpdated:
+                    type: string
+                    description: |-
+                      Only satisfy the Search if the Beneficiary's `last_updated` Date falls within a specified _DateRange_.
+                      A _DateRange_ can be defined by providing less than `lt` and/or greater than `gt` values.
+                      This parameter can be included in a request one or more times.
+
+                      Examples:
+                        - `_lastUpdated=gt2023-01-02&_lastUpdated=lt2023-05-01` defines a range between two provided dates
+                        - `_lastUpdated=gt2023-01-02` defines a range between the provided date and today
+                        - `_lastUpdated=lt2023-05-01` defines a range from the earliest available records until the provided date
+                    example: "gt2023-01-02"
+                  excludeSAMHSA:
+                    type: string
+                    description: |-
+                      The _Substance Abuse and Mental Health Services Administration_ (SAMHSA)
+                      is the agency within the U.S. Department of HHS that leads public health efforts to advance the behavioral health of the nation.
+                      Setting this flag to _true_, modifies the request to filter out all SAMSHA-related claims from the response.
+                      _ClaimResponse_ doesn't contain any SAMHSA potential data. Due to the 1:1 relationship between _ClaimResponse_
+                      and _Claim_, any ClaimResponse resource will be excluded when the associated Claim resource contains SAMHSA data.
+
+                      Example:
+                        - `excludeSAMHSA=true`
+                    example: true
+                  identifier:
+                    type: string
+                    includeTaxNumbers: |-
+                      Setting this flag to _true_, provides tax number in the EOB transformed data for the response.
+
+                      Example:
+                        - `includeTaxNumbers=true`
+                    example: true
+                  isHashed:
+                    type: string
+                    description: |-
+                      Setting this flag to _true_, provides tax number in the EOB transformed data for the response.
+
+                      Example:
+                        - `isHashed=true`
+                              example: "ABCD"
```

Comment:
This has both some formatting and content challenges... I don't think the content-related issues are limited to this specific block, either. Hmm.

Reply:
I will work on the formatting, I will updating the existing doc as well because they all follow the same formatting. The content is copied from the recent `OpenAPI.yaml`, should I have a different source?

Reply:
Right–it must be inaccurate in the source material you're referring to. I gather `isHashed` is a temporary input that's used with an `mbi` input to signal whether the input should be treated as hashed or plaintext, e.g. `mbi=1234&isHashed=false`.

Again, I admit that I haven't looked too closely at the implementation... can you also document what the default behavior is? If `isHashed` is omitted, does the API treat the inputs as hashed or unhashed by default?

Reply:
looks like `isHashed` is required. without specifying `isHashed=false ` It doesn't return anything but with returns all the values expected.

```
{
  "resourceType": "Bundle",
  "id": "12345",
  "meta": {
    "lastUpdated": "2024-09-10"
  },
  "type": "searchset",
  "total": 0,
  "link": [
    {
      "relation": "self",
      "url": "https://localhost:6500/v2/fhir/Claim?mbi=12345"
    }
  ]
}
```

Reply:
default is as setting `isHashed=true`